content/Software Support/Upload/Fix-udev-rules-on-Linux.md

```diff
@@ -0,0 +1,145 @@
+---
+title: "Fix udev rules on Linux"
+---
+
+On Linux, missing udev[^1] rules can result in failed uploads, as Arduino IDE (and other development tools) will not be able to access the board when it resets to bootloader mode.
+
+The board package typically includes a script that is run during installation, which sets the appropriate udev rules. If this script didn't run correctly, or if the rules were removed, they can be configured by running the script manually:
+
+[^1]: <https://manpages.ubuntu.com/manpages/xenial/man7/udev.7.html>
+
+<table>
+<thead>
+  <tr>
+    <th>Boards</th>
+    <th>Related error output</th>
+    <th>Instructions</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td>Nano RP2040 Connect</td>
+    <td><code>Failed uploading: uploading error: exit status 1</code></td>
+    <td rowspan="4"><a href="#mbed-os">Set udev rules for Arduino Mbed OS boards</a></td>
+  </tr>
+  <tr>
+    <td>GIGA R1 WiFi</td>
+    <td>
+      <code>dfu-util: Cannot open DFU device 2341:0366 found on devnum 24 (LIBUSB_ERROR_ACCESS)</code><br>
+      <code>dfu-util: No DFU capable USB device available</code><br>
+      <code>Failed uploading: uploading error: exit status 74</code>
+    </td>
+  </tr>
+  <tr>
+    <td>Nicla Sense ME</td>
+    <td>
+      <code>Error: unable to open CMSIS-DAP device 0x2341:0x60</code><br>
+      <code>Error: unable to find a matching CMSIS-DAP device</code><br>
+      <code>Failed uploading: uploading error: exit status 1</code>
+    </td>
+  </tr>
+  <tr>
+    <td>
+      Portenta H7<br>
+      <i>Portenta H7 Lite</i><br>
+      <i>Portenta H7 Lite Connected</i>
```

Comment:
Is there a reason for the italic here? (Maybe users should be aware of different rules or notes for the Portenta H7 Lite, Portenta H7 Lite Connected). If not, it should be removed not to create confusion